driver/controller.go

```diff
@@ -94,30 +92,57 @@ func newControllerService(client *v3.Client, nodeMeta *nodeMetadata) controllerS
 // This function is idempotent.
 func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
 	klog.V(4).Infof("CreateVolume")
-	// TODO(multizone cluster) use req.AccessibilityRequirements,
-	// To create block storage volume in the right zone.
 
-	// TODO(multizone cluster) fetch all zone
-	volumes, err := d.client.ListBlockStorageVolumes(ctx)
+	zones, err := d.client.ListZones(ctx)
 	if err != nil {
-		klog.Errorf("create block storage volume list: %v", err)
+		klog.Errorf("create block storage volume list zones: %v", err)
 		return nil, err
 	}
 
-	// Make the call idempotent since CreateBlockStorageVolume is not.
-	for _, v := range volumes.BlockStorageVolumes {
-		if v.Name == req.Name {
-			return &csi.CreateVolumeResponse{
-				Volume: &csi.Volume{
-					VolumeId: exoscaleID(d.zoneName, v.ID),
-					// API reply in bytes then send it without conversion
-					CapacityBytes:      v.Size,
-					AccessibleTopology: newZoneTopology(d.zoneName),
-				},
-			}, nil
+	// Fetch all zones volumes
```

Comment:
Maybe only for the `createVolume` (not for the list..etc), only use the zone returned by the`AccessibilityRequirements` topology